CONTRIBUTING.md

```diff
@@ -82,7 +82,7 @@ public class MyCmdlet : SPOWebCmdlet
 #endif
 ```
 
-If only parts of a cmdlet require different behaviour based upon the different version of the SDK, you are recommended to use the #ONPREMISES or other available preprocessor variable throughout your code to exclude or include certain code.
```

Comment:
This is correct English. Why change this to American English?

Reply:
The original spelling is actually correct for me as well (🇨🇦) the the "neutral" language of .Net and the Microsoft docs is en-us, so I switched all the british spellings to americanized versions

Reply:
https://docs.microsoft.com/en-ca/style-guide/word-choice/use-us-spelling-avoid-non-english-words